core/src/main/scala/org/apache/spark/rdd/DoubleRDDFunctions.scala

```diff
@@ -74,6 +74,22 @@ class DoubleRDDFunctions(self: RDD[Double]) extends Logging with Serializable {
   }
 
   /**
+   * Compute the population standard deviation of this RDD's elements.
+   * @since 2.1.0
```

Comment:
It's valid, though I don't think we generally use them. I would just use the `@Since` tag along in Scala, since it is processed differently on purpose.